### PR TITLE
Set-up linters and add required files for Hello Microverse Project

### PR DESCRIPTION
**Pull request by nmeziany**
In this project I:
-Set up linters for html and css.
-Created html and css files.
-Ignored files for tests in .gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## Built With
+
+- HTML and CSS
+-VSCode
+
+
+## Authors
+
+- GitHub: [@nmeziany][https://github.com/nmeziany/PortfolioSkeleton]
+- linkedIn: [https://www.linkedin.com/in/noureddine-meziany/]
```
Comment (Whoistolu) [inline README.md]: - [x] [REQUIRED] You have done well implementing the README file however, it is not that descriptive yet. You left out the project description, MIT license, and the project image. Please see [Microverse readme-template here](https://github.com/microverseinc/readme-template) for more explanation.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-*txt
\ No newline at end of file
+*txt
+test
```
Comment (Whoistolu) [inline .gitignore]: - [x] [REQUIRED] One of the requirements states that test.md file and `node_modules` directory should be ignored by `.gitignore`. Please go-ahead to add them.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @nmeziany 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] [REQUIRED] You have done great work with your commit messages, however, here at Microverse, we use passive statements in commit messages and not past tense. This ensures more professionalism. See below:
### Your commit message
your-commits
### Recommended commit messages
proper-commit-messages

- [x] [REQUIRED] kindly ensure that your Pull Request has a descriptive title telling you what changes are inside. This is a good professional practice.
Currently, your PR title reads Setlinter which does not describe the changes you have made in the branch.

- [x] [REQUIRED] Please note that the branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section') and not setlinter as your branch reads. This also is a good practice.

_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A
Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .gitignore (hunk 2 above)
Comment (nmeziany) [reply .gitignore]: mode_modules and test.md files added to .gitignore file

File: README.md (hunk 1 above)
Comment (nmeziany) [reply README.md]: Modifications done for README file

**Review by rustam-tolipov — APPROVED** [review]
## Hi @nmeziany 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**